modules/ROOT/pages/getting-started.adoc

```diff
@@ -7,63 +7,93 @@ xref:glossary:index.adoc[glossary].
-individual according to users' roles and the permissions defined by 
+individual according to their roles and the permissions defined by 
 link:https://konflux-ci.dev/architecture/ADR/0011-roles-and-permissions.html[those roles]. The tenant namespaces can be either
 for an individual or a team.
 //TODO: Document the process for getting access to/creating new namespaces
 
-You can grant, modify, or remove other users access to any namespace for which you have the `admin` role. Each
-namespace's admins have the responsibility to ensure that appropriate access is granted to users. If users are
-over-privileged, they may have access to any added secrets or to perform other unintended actions.
+In {ProductName}, you operate in a tenant namespace which is scoped to your
+team. One team can span multiple namespaces if needed, each with many
+applications with components. Namespaces should NOT be shared by many teams.
+
+In your tenant namespace, you will xref:building:creating.adoc[create
+Components and Applications], run the xref:building:index.adoc[build
+PipelineRuns] that are defined in your git repositories, view and iterate on
+the xref:testing:index.adoc[results of your IntegrationTestScenarios], and
+xref:releasing:index.adoc[create Releases] to release specific Snapshots
+according to specific ReleasePlans.
 
 ==== Managed namespace
-Sometimes users will need to perform actions that require credentials in someone else's control. Since admins
-in a tenant namespace can see all secrets, these actions need to be performed via separate managed namespaces.
-The primary interaction mode between tenant and managed namespaces is to create a Release referencing a specific
-Snapshot which will trigger a specific pipeline in the managed namespace.
+These are namespaces where a managed environment team manages release pipelines
+and credentials for your organization.
+
+The primary interaction mode between tenant and managed namespaces is to create
+a Release in a tenant namespace referencing a specific Snapshot which will
+trigger a specific release pipeline in the managed namespace. See
+xref:releasing:index.adoc[release documentation] for more.
+
+=== Custom Resource (CR)
+In Kubernetes, a
+link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[Custom
+Resources (CR)] is an extension of the Kubernetes API.
+
+All {ProductName} APIs are implemented as Kubernetes CRs. If
+you're familiar with common Kubernetes resources such as Pods and Deployments,
+you'll find that {ProductName} resources appear on cluster in the same way:
+Applications, Components, Snapshots, and PipelineRuns.
+
+A consequence of this fact is that commonplace Kubernetes client tools such as
+link:https://kubernetes.io/docs/reference/kubectl/[kubectl] understand and can
+work with {ProductName} APIs.
 
 === Component
 A Component CR describes the properties for an OCI artifact including the git repository where the artifact is from,
```

Comment:
It might be just me, but I wonder if OCI artifact is a term well-known enough that it does need any further explanation. We could add a tiny explanation in the glossary and link the references there (or link them to an official definition).

Reply:
Updated in 57d76dc.